### PR TITLE
Cyborgsecurity hunter v1 image update

### PR DESCRIPTION
**Pull request by nbyt3**
Required items, please complete
   
   Change(s):
   - Updated logo name to match the maintemplate file

   Reason for Change(s):
   - 
Screenshot 2023-11-17 083556

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/Cyborg Security HUNTER/Data/Solution_CyborgSecurity_HUNTER.json
Comment (v-dvedak) [inline Solutions/Cyborg Security HUNTER/Data/Solution_CyborgSecurity_HUNTER.json:1]: Please check the logo path. It is missing the file extension.

File: Solutions/Cyborg Security HUNTER/ReleaseNotes.md
Comment (v-dvedak) [inline Solutions/Cyborg Security HUNTER/ReleaseNotes.md:1]: Please check the order of release notes. The latest one should be at the top.

File: Solutions/Cyborg Security HUNTER/Data/Solution_CyborgSecurity_HUNTER.json
Comment (nbyt3) [reply Solutions/Cyborg Security HUNTER/Data/Solution_CyborgSecurity_HUNTER.json:1]: Fixed location

File: Solutions/Cyborg Security HUNTER/ReleaseNotes.md
Comment (nbyt3) [reply Solutions/Cyborg Security HUNTER/ReleaseNotes.md:1]: Updated the release notes

**Comment by v-dvedak** [comment]
Please check the version at all the places. It should be 3.0.1 at all the places including - /Data/Solution_CyborgSecurity_HUNTER.json, mainTemplate.json and zip.

**Comment by nbyt3** [comment]
> Please check the version at all the places. It should be 3.0.1 at all the places including - /Data/Solution_CyborgSecurity_HUNTER.json, mainTemplate.json and zip.

@v-dvedak  thank you for the feedback. I updated the version number in all mentioned locations

**Comment by v-dvedak** [comment]
Hi @nbyt3, thank you for taking a look at the feedback. I see an extra folder (__MACOSX) under the 3.0.1.
As per my understanding since the 3.0.0 version of the solution didn't go Live ( due to certification rejection) so you can use the same version (3.0.0). Just a request that you repackage solution with 3.0.0 to ensure the changes reflected correctly at all the places.

**Comment by nbyt3** [comment]
@v-dvedak I just fixed the zip to remove the _MACOSX folder and confirm the files in the zip are correct

**Comment by v-atulyadav** [comment]
Hi @nbyt3, 
Please repackage this solution with version 3.0.0 instead of 3.0.1. Also release note date format is incorrect.
image

**Comment by nbyt3** [comment]
@v-atulyadav I updated the suggested files and re-package with version 3.0.0

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**